### PR TITLE
added space cadet clothing Refs: https://github.com/gitcoinco/web/iss…

### PR DESCRIPTION
**Pull request by marnibrewster**
…ues/1663

##### Description

A new avatar clothing option. A space cadet outfit as per @owocki request. I know there is a spacesuit (like NASA) in the works with a helmet. I thought a retro 70s style spacesuit would be fun. Please disregard if this is not wanted or needed, this was an easy way for me to get the bitcoin app running locally and learn the contributing docs. I thought this clothing might pair well with the spacesuit helmet also being created. 

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)
- [x] ran svgo on svg files 

##### Testing

I ran `npm run eslint:fix; npm run stylelint:fix; isort -rc --atomic `

##### Refers/Fixes

  Refs: #1663
Inspiration: 
70scadet-01
60scadet-01
Screen capture of builder: 
gitcoin_avatar
Screen capture of new profile: 
screen shot 2018-09-20 at 7 24 25 pm

### CODE REVIEW

File: app/app/local.env [hunk 1]
```diff
@@ -17,7 +17,7 @@ DEBUG=on
 ################################################################################
 # Be VERY CAREFUL when changing this setting.  You don't want to accidentally
 # spam a bunch of github notifications :)
-ENABLE_NOTIFICATIONS_ON_NETWORK=
+ENABLE_NOTIFICATIONS_ON_NETWORK=None
```
Comment (mbeacom) [inline app/app/local.env]: Can you drop the changes in this file?

File: app/avatar/templates/clothing/spacecadet.txt [hunk 2]
```diff
@@ -0,0 +1 @@
...2,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/><path d="M862,1335l42-42,42,42-42,42Z" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="6.9"/></g><path d="M208,1254l-9,142m492-142,5,142" transform="translate(60 -829)" fill="none" stroke="#333" stroke-linecap="round" stroke-width="12.52"/><path d="M450,1079a469,469,0,0,1,184,38l117,50a156,156,0,0,1,95,143v79H54v-79a156,156,0,0,1,95-143l117-50a469,469,0,0,1,184-38Zm0-27a497,497,0,0,0-195,40l-117,50A183,183,0,0,0,27,1310v106H872V1310a183,183,0,0,0-111-168l-117-50a496,496,0,0,0-195-40Z" transform="translate(60 -829)" fill="#333"/><path d="M548,1084c0,1,0,20,0,21,0,54-45,97-102,97s-102-44-102-97v-21" transform="translate(60 -829)" fill="none" stroke="#333" stroke-width="11.05" opacity="0.15" style="isolation:isolate"/><polygon points="446 571 333 261 100 124 242 261 72 264 347 572 446 571" fill="#ffcc3b" stroke="#333" stroke-miterlimit="10" stroke-width="26.97"/><polygon points="574 570 687 261 920 124 778 261 948 264 673 572 574 570 574 570" fill="#ffcc3b" stroke="#333" stroke-miterlimit="10" stroke-width="26.97"/></g></svg>
```
Comment (mbeacom) [inline app/avatar/templates/clothing/spacecadet.txt]: Can you modify the fill colors in this `.txt` to take primary_color similarly to the existing `.txt` templates?

**Comment by thelostone-mc** [comment]
@marnibrewster could we rebase and resolve conflicts ? else LGTM

File: app/avatar/utils.py [hunk 3]
```diff
@@ -67,11 +67,41 @@ def get_avatar_context():
...
+                '25E899', '9AB730', '00A55E', '3FCDFF', '3E00FF', '8E2ABE', 'D0021B', 'F9006C', 'FFCE08', 'F8E71C',
+                '15003E', 'FFFFFF'
+            )
+        }],
         },
                      {
                          'name': 'Clothing',
                          'title': 'Pick your clothing',
-                         'options': ('cardigan', 'hoodie', 'knitsweater', 'plaid', 'shirt', 'shirtsweater')
+                         'options': ('cardigan', 'hoodie', 'knitsweater', 'plaid', 'shirt', 'shirtsweater', 'spacecadet')
```
[the remark this answers is not in the document]
Comment (mbeacom) [reply app/avatar/utils.py]: Just need to fix this line length and it should be good to go.

**Comment by thelostone-mc** [comment]
Thanks @marnibrewster for the crazy space cadet design :D

**Comment by owocki** [comment]
woah this is so cool!  paying out now

**Comment by owocki** [comment]
@marnibrewster any interest in doing a few more pieces with us?  I really `<3` [my space cadet outfit](https://twitter.com/owocki/status/1047334765664194560)!

**Comment by thelostone-mc** [comment]
@marnibrewster Dresses for women ? ^_^

**Comment by marnibrewster** [comment]
Thanks all! I'm so glad you liked it! I was intimidated to get started and worried I would mess something up. I'm checking out October's updated requests now and will get the wheels spinning!

**Comment by owocki** [comment]
@marnibrewster yayyyyyy 👩‍🎨

**Pull request merged**